### DISCUSSION TITLE
Why GACSTR+ is the default algorithm for tables?

### DISCUSSION THREAD

**Discussion by gadavidd** [chocoteam/choco-solver#1073] [General]
I'm working with table propagators and then I realized that in `IIntConstraintFactory` class, the default configuration is `GACSTR+` algorithm for feasible tuples. 
At first, I thought it was due to the memory optimization used, once the specific options are `STR2+` and `CT+`. But, I'm not sure about it.

**Comment by cprudhom** [comment]
The conditions were extracted from empirical evaluations, few years from now.
I suppose that this could be re-evaluated.
